erpnext/patches/v12_0/set_published_in_hub_tracked_item.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	frappe.reload_doc("Hub Node", "doctype", "Hub Tracked Item")
+	if frappe.db.a_row_exists("Hub Tracked Item"):
+		return
+
+	frappe.db.sql('''
+		Update `tabHub Tracked Item`
+		SET published = 1
+	''')
```

Comment:
If there is a row in Hub Tracked Item then the SQL statement won't run. Is this the intended behavior?